### PR TITLE
Add all Python SDK packages to top-level init file

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

File: sdk/python/pulumi_kubernetes/__init__.py [hunk 1]
```diff
@@ -26,6 +26,9 @@
     "scheduling",
     "settings",
     "storage",
+    "helm",
```
Comment (justinvp) [inline sdk/python/pulumi_kubernetes/__init__.py]: Drive-by review: Nit: It looks like all the other lines are sorted alphabetically.

Comment (lblackstone) [reply sdk/python/pulumi_kubernetes/__init__.py]: Yeah, this file is generated from a mustache template, and these packages weren't part of the templated list. It's nontrivial to add them, so I just put them at the end.

Comment (justinvp) [reply sdk/python/pulumi_kubernetes/__init__.py]: Ah, right. Sorry for the noise!
